src/js/server/utils.js

```diff
@@ -108,12 +132,36 @@ gpii.express.user.utils.createNewUser = function (that, userData) {
     return promiseTogo;
 };
 
+/**
+ * gpii.express.user.utils.verifyPassword function
```

Comment:
What value do these add versus the code itself?  Are they useful in your docs generation?